Containerfile.external-dns-operator-catalog

```diff
@@ -20,7 +20,7 @@ COPY --from=drift /app/Dockerfile.catalog .
 ENTRYPOINT ["/bin/opm"]
 
 # Copy the EDO FBC catalog (OCP version-based)
-COPY catalog/v4.21/catalog.yaml /configs/external-dns-operator/catalog.yaml
+COPY catalog/v4.19/catalog.yaml /configs/external-dns-operator/catalog.yaml
```

Comment:
This change is not needed anymore. The test is supposed to be run on 4.19 as ITS is not linked to 4.19 FBC.